.changeset/fifty-snails-do.md

```diff
@@ -0,0 +1,5 @@
+---
+'@kaizen/components': patch
+---
+
+add Split Button Replacement story and docs to Button & Menu sections
```

Comment:
Since this is updating the documentation, do we need to release a patch?

Reply:
Good catch, I changed it to patch the docs since it updates content there but the files live in the components dir